src/app/hooks/useClickTrackerHandler/index.jsx

```diff
@@ -33,107 +31,186 @@ const useClickTrackerHandler = (props = {}) => {
+              optimizely.close();
             }
+            window.location.assign(nextPageUrl);
           }
         }
       }
-    },
-    [
-      trackingIsEnabled,
-      clicked,
-      campaignID,
-      componentName,
-      pageIdentifier,
-      platform,
-      preventNavigation,
-      producerId,
-      producerName,
-      service,
-      statsDestination,
-      url,
-      advertiserID,
-      format,
-      optimizely,
-      optimizelyMetricNameOverride,
-      detailedPlacement,
-      useReverb,
-    ],
-  );
+    }
+  }, Object.keys(beaconProps));
+};
+
+export const LITE_TRACKER_FUNCTION = 'liteTrackerFunction';
+
+export const liteTrackingScript = () => {
+  return `function ${LITE_TRACKER_FUNCTION}(event, atiURL){
+      event.stopPropagation();
+      event.preventDefault();
+
+      var nextPageUrl = event.currentTarget.href;
+     
+      const { screen: {width, height, colorDepth, pixelDepth}, innerWidth, innerHeight } = window;
+      const now = new Date();
+      const hours = now.getHours();
+      const mins = now.getMinutes();
+      const secs = now.getSeconds();
+    
+      // COOKIE SETTINGS
+      const cookieName = 'atuserid';
+      const expires = 397; // expires in 13 months
+      let cookiesForPage = \`; $\{document.cookie\}\`;
+      let atUserIdCookie = cookiesForPage.split(\`; $\{cookieName\}=\`);
+      let atUserIdValue = null;
+
+      if (atUserIdCookie.length === 2){
+        const cookieInfo = atUserIdCookie.pop().split(';').shift();
+
+        if(cookieInfo){
+          let decodedCookie = decodeURI(cookieInfo);
+          const user = JSON.parse(decodedCookie);
+          atUserIdValue = user.val;
+        }
+      } 
+
+      if(!atUserIdValue && crypto.randomUUID){
+          atUserIdValue =  crypto.randomUUID();
+      }
+
+      const stringifiedCookieValue = JSON.stringify({val: atUserIdValue});
+      if(atUserIdValue){
+        document.cookie = \`$\{cookieName\}=$\{stringifiedCookieValue\}; path=/; max-age=$\{expires\};\`;
+      }
+      
+      const rValue = [
+        width || 0,
+        height || 0,
+        colorDepth || 0,
+        pixelDepth || 0,
+      ].join('x')
+      
+      const reValue = [innerWidth || 0, innerHeight || 0].join('x');
+
+      const hlValue = [hours, mins, secs].join('x');
+
+      let clientSideAtiURL = atiURL
+                                .concat('&', 'r=', rValue)
+                                .concat('&', 're=', reValue)
+                                .concat('&', 'hl=', hlValue)
+      
+      if (navigator.language) {
+        clientSideAtiURL = clientSideAtiURL.concat('&', 'lng=', navigator.language)
+      }                     
+       
+      if (atUserIdValue) {
+        clientSideAtiURL = clientSideAtiURL.concat('&', 'idclient=', atUserIdValue)
+      }     
+
+      sendBeaconLite(clientSideAtiURL);
+
+      window.location.assign(nextPageUrl);
+  }`;
+};
+
+export const constructLiteSiteURL = props => {
```

Comment:
Can we rename this so that it's more representative of what it does?
```suggestion
export const constructLiteSiteATIURL = props => {
```